src/javascript/app_2/Stores/Modules/Trading/Constants/validation_rules.js

```diff
@@ -31,6 +35,19 @@ const getValidationRules = () => ({
             ['req'    , { message: localize('Duration is a required field.') }],
         ],
     },
+    start_time: {
+        rules: [
+            ['custom' , { func: (value, options, store) => store.contract_start_type === 'spot' || isTimeValid(value), message: localize('Please enter the start time in the format "HH:mm".') }],
+            ['custom' , { func: (value, options, store) => {
+                if (store.contract_start_type === 'spot') return true;
+                if (!isTimeValid(value)) return false;
+                const start_moment       = toMoment(store.start_date);
+                const start_moment_clone = start_moment.clone();
+                const [h, m] = value.split(':');
+                return isSessionAvailable(store.sessions, start_moment_clone.hour(h).minute(m), start_moment);
+            }, message: localize('Start time cannot be in the past.') }],
+        ],
```

Comment:
@kellybinary It should have a custom rule to check that the value of hour and minute are respectively between 0-23 and 0-59.

Reply:
@morteza-binary done :)